src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -254,7 +255,13 @@ export class HuiDialogEditCard extends LitElement {
 
   private _handleCardPicked(ev) {
     const config = ev.detail.config;
-    if (this._params!.entities && this._params!.entities.length > 0) {
+    if (
+      this._params!.entities &&
+      this._params!.entities.length > 0 &&
+      ((Object.keys(config).includes("entities") &&
+        config.entities.length === 0) ||
+        (Object.keys(config).includes("entity") && !config.entity))
```

Comment:
We should always overrule if there are entities passed with the params.

Reply:
I should have removed this anyway. It was before I stopped trying to add so much to the PR. this change was to fix an issue I was seeing when params had entities. Will address in a future PR